mason.sh

```diff
@@ -119,7 +119,16 @@ elif [ ${MASON_PLATFORM} = 'linux' ]; then
     fi
 
 elif [ ${MASON_PLATFORM} = 'android' ]; then
-    export MASON_ANDROID_ABI=${MASON_ANDROID_ABI:-arm-v7}
+    case "${MASON_PLATFORM_VERSION}" in
+        arm-v5-9) export MASON_ANDROID_ABI=arm-v5 ;;
+        arm-v7-9) export MASON_ANDROID_ABI=arm-v7 ;;
+        arm-v8-21) export MASON_ANDROID_ABI=arm-v8 ;;
+        x86-9) export MASON_ANDROID_ABI=x86 ;;
+        x86_64-21) export MASON_ANDROID_ABI=x86 ;;
+        mips-9) export MASON_ANDROID_ABI=mips ;;
+        mips64-21) export MASON_ANDROID_ABI=mips64 ;;
+        *) export MASON_ANDROID_ABI=${MASON_ANDROID_ABI:-arm-v7}
+    esac
 
```

Comment:
Could also be accomplished with `if [[ $${MASON_PLATFORM_VERSION} =~ "mips-" ]]; then export MASON_ANDROID_ABI=mips .. `